service/notifications.go

```diff
@@ -114,5 +114,11 @@ func (producer *AvailabilityStatusNotifier) EmitAvailabilityStatusNotification(a
 }
 
 func EmitAvailabilityStatusNotification(accountNumber string, emailNotificationInfo *m.EmailNotificationInfo) error {
+	l.Log.Infof("Publishing message as status notification for source(ID:%s, tenant_id: %s), status changed from: %s to %s",
+		emailNotificationInfo.SourceID,
+		emailNotificationInfo.TenantID,
+		emailNotificationInfo.CurrentAvailabilityStatus,
+		emailNotificationInfo.PreviousAvailabilityStatus)
+
```

Comment:
I'd like to open a discussion on this. I've been following the following format lately:

```go
`[tenant_id: %d][source_id: %s] Publishing status notification. Status changed from "%s" to "%s"`
```
I like wrapping values around `[]` and `""` so that they're easily identifiable once you're watching the logs. What do you think?